src/value.rs

```diff
@@ -344,9 +394,8 @@ impl ValueCommitment {
     ///
     /// [concretehomomorphiccommit]: https://zips.z.cash/protocol/nu5.pdf#concretehomomorphiccommit
     #[allow(non_snake_case)]
-    pub fn derive(value: ValueSum, rcv: ValueCommitTrapdoor) -> Self {
+    pub fn derive(value: ValueSum, rcv: ValueCommitTrapdoor, asset: AssetBase) -> Self {
         let hasher = pallas::Point::hash_to_curve(VALUE_COMMITMENT_PERSONALIZATION);
-        let V = hasher(&VALUE_COMMITMENT_V_BYTES);
         let R = hasher(&VALUE_COMMITMENT_R_BYTES);
```

Comment:
_From str4d ([2542846450](https://github.com/zcash/orchard/pull/471#discussion_r2542846450))_

```suggestion
        let V = asset.cv_base();
        let R = hasher(&VALUE_COMMITMENT_R_BYTES);
```